src/Raven.Server/Documents/PeriodicBackup/PeriodicBackupRunner.cs

```diff
@@ -985,8 +985,11 @@ public RunningBackup OnGoingBackup(long taskId)
 
         public string TombstoneCleanerIdentifier => "Periodic Backup";
 
-        public Dictionary<string, long> GetLastProcessedTombstonesPerCollection()
+        public Dictionary<string, long> GetLastProcessedTombstonesPerCollection(ITombstoneAware.TombstoneType tombstoneType)
         {
+            if (tombstoneType != ITombstoneAware.TombstoneType.Documents)
```

Comment:
For replication and backup I don't think you care what type it is. You just need to know the last processed etag overall.
Right now we might delete dead segments without backing them up.